lib/razor/config.rb

```diff
@@ -24,42 +24,53 @@ class Config
 
     def initialize(env, fname = nil, defaults_file = nil)
       @values = {}
-      # Load defaults first.
+      # Find the defaults file.
+      # Priority: 1) environment variable
+      # 2) default path in /opt
+      # 3) root directory
       defaults_file ||= ENV["RAZOR_CONFIG_DEFAULTS"] ||
           (File.file?('/opt/puppetlabs/server/apps/razor-server/config-defaults.yaml') and '/opt/puppetlabs/server/apps/razor-server/config-defaults.yaml') ||
           File::join(File::dirname(__FILE__), '..', '..', 'config-defaults.yaml')
-      begin
+
+      # Find the user-maintained config.yaml
+      # Priority: 1) environment variable
+      # 2) default path in /etc
+      # 3) root directory
+      fname ||= ENV["RAZOR_CONFIG"] ||
```

Comment:
I think it'd be clearer if this had a less generic name, maybe: `config_defaults_file` and config_user_file`?